microbenchmarking_test.py

```diff
@@ -33,4 +33,6 @@ def test_perf_simple_query(self):
                 specific_tested_stats={"perf_simple_query_result": json.loads(output)},
                 doc_id_with_timestamp=True)
             if self.create_stats:
-                PerfSimpleQueryAnalyzer(self._test_index, self._es_doc_type).check_regression(self._test_id)
+                is_gce = bool(self.params.get('cluster_backend') == 'gce')
```

Comment:
Result of expression a == b , is bool, so no need to explicitly

Reply:
fixed